### ISSUE TITLE
'PlinkReader' object has no attribute 'get_all_genotypes'

### ISSUE THREAD

**Issue by daniel-munro** [broadinstitute/tensorqtl#29]
get_all_genotypes() is called by load_genotypes() and called in README.md, but is not defined, giving the error "'PlinkReader' object has no attribute 'get_all_genotypes'".

**Comment by francois-a** [comment]
Hi,
Thanks for catching this (fixed in 7432e3d6aa5e947da610b925ffc1788b4c9113f7). The interface for loading all genotypes was simplified to:
```
pr = genotypeio.PlinkReader(plink_prefix_path)
genotype_df = pr.load_genotypes()
```

**Comment by daniel-munro** [comment]
Thanks!
